pkg/web/src/app/queries/fetchHelpers.ts

```diff
@@ -5,7 +5,9 @@ import { History, LocationState } from 'history';
 import { useClientInstance } from '@app/client/helpers';
 import { KubeResource } from '@konveyor/lib-ui';
 import { IKubeResponse, IKubeStatus } from '@app/client/types';
+import { IEnvVars } from '@app/common/types';
 
+export const ENV: IEnvVars = window['_env'] || {};
```

Comment:
Instead of declaring a new `ENV` in each file where it is used, we should continue to import the centralized `ENV` that is exported by `pkg/web/src/app/common/constants.ts`. You can see where this is already done in `ForkliftAboutModal`. You can import it like this (the `@app` import path is an alias for `pkg/web/src/app` in our tsconfig):

```ts
import { ENV } from '@app/common/constants';
```

Aside from saving you from importing `IEnvVars`, the reason for this is just in case we ever need/want to modify the way these environment variables are provided to / consumed by the UI, since the `window['_env']` global is a bit of a hack. We want to keep that hack isolated 😄